tests/functional/spec/supplementary_data_interstitial/supplementary_data_with_interstitial_and_calculated_summary.spec.js

```diff
@@ -0,0 +1,136 @@
+import { click, assertSummaryTitles } from "../../helpers";
```

Comment:
We don't name specs based on the schema they open; instead, we focus on the functionality being tested in the spec. For example, `conditional_question_title` opens `test_titles_radio_and_checkbox.json`, so it should be renamed to reflect what is being tested in this schema. Are these specs grouped thematically? The title should clearly indicate what is being tested.